Functions/xASL_bids_MergeNifti.m

```diff
@@ -187,7 +187,7 @@
 	end
 		
 	% If GE is not identified or ImageType field doesn't exist, then exits
-	if ~regexpi(varManufacturer,'GE') || ~isfield(jsonPar,'ImageType')
+	if isempty(regexpi(varManufacturer,'GE')) || ~isfield(jsonPar,'ImageType')
 		return;
```

Comment:
I guess he needs to skip the rest of the processing pipeline and an error wouldn't be helpful. But maybe we should add a printf or a warning?

Reply:
Haha - you are right :) Though in this case, the return is used to break a for-cycle that goes through all files. So, we want to interrupt the execution of the for-cycle to stop loading more JSONs if the conditions are met to save processing time. At the same time, we cannot put this condition elsewhere, because we would have to pre-read all the JSONs.

Altogether - you are right, but to get rid of the `return` here, we would have to move this `if` elsewhere outside of the `for` cycle and this would result in more time-consuming code. So in this case, the `return` is justified and hard to replace.